modules/KIWILinuxRC.sh

```diff
@@ -10493,6 +10494,10 @@ function pxePartitionInputGeneric {
             partID=fd
             pname=lxroot
         fi
+        if [ $partID = "ef" ] && [ ! -z "$RAID" ];then
```

Comment:
My test fails on this: [ ! -z "$RAID" ]

Reply:
Damn sorry, just realized I did not push this change last Friday, this was supposed to be 

```bash
if [ $partID = "ef" ] && [ "$partMount" = "/boot/efi" ];then
```